File: CloudBridge-WD/src/main/java/de/r3pt1s/CloudBridge/network/protocol/packet/DispatchCommandPacket.java
// This class was created by r3pt1s
package de.r3pt1s.CloudBridge.network.protocol.packet;

public class DispatchCommandPacket extends Packet {

    public Object server = "";
    public Object commandLine = "";

    @Override
    public void encode() {
        super.encode();
        put(server);
        put(commandLine);
    }

    @Override
    public void decode() {
        super.decode();
        server = get();
        commandLine = get();
    }

    @Override
    public Integer getId() {
        return ID_DISPATCH_COMMAND;
    }

    public static DispatchCommandPacket create(String server, String commandLine) {
        DispatchCommandPacket pk = new DispatchCommandPacket();
        pk.server = server;
        pk.commandLine = commandLine;
        return pk;
    }
}

File: CloudSystem/src/Cloud/network/protocol/packet/DispatchCommandPacket.php
<?php

namespace Cloud\network\protocol\packet;

class DispatchCommandPacket extends Packet {

    public string $server = "";
    public string $commandLine = "";

    public function encode(): void {
        parent::encode();
        $this->put($this->server);
        $this->put($this->commandLine);
    }

    public function decode(): void {
        parent::decode();
        $this->server = $this->get();
        $this->commandLine = $this->get();
    }

    public function getId(): int {
        return self::ID_DISPATCH_COMMAND;
    }

    public static function create(string $server, string $commandLine): self {
        $pk = new self;
        $pk->server = $server;
        $pk->commandLine = $commandLine;
        return $pk;
    }
}